src/main/kotlin/no/nav/syfo/util/VarselUtil.kt

```diff
@@ -25,6 +27,22 @@ class VarselUtil(private val databaseAccess: DatabaseInterface) {
             .filter { it.type == varselType.name }
     }
 
+    fun isVarselInEffect(fnr: String, varselType: VarselType): Boolean {
+        val utsendtVarsel = databaseAccess.fetchUtsendtVarselByFnr(fnr)
+            .filter { it.type == varselType.name }
+
+        if (utsendtVarsel.isEmpty()) {
+            return false
+        }
+
+        utsendtVarsel.sortedBy { it.utsendtTidspunkt }
+        val sisteUtsendeVarsel = utsendtVarsel.last()
+        val sisteGangVarselBleUtsendt = sisteUtsendeVarsel.utsendtTidspunkt.toLocalDate()
+        val sisteDagISykefravaeret = sisteGangVarselBleUtsendt.plusWeeks(13L)
+        val dagsDato = LocalDate.now()
+        return dateIsInInterval(dagsDato, sisteGangVarselBleUtsendt, sisteDagISykefravaeret)
```

Comment:
Det var sånn at Lisa skal verifisere denne logikken? Det ser jo greit ut for meg, men hvis jeg skjønner riktig så vil den sykmeldte får varsel på nytt hvis hen sender inn sykmelding etter 52 uker.

Reply:
@kaja-nav Det stemmer. I så fall vil vedkommende være på slutten av forløpet dersom det ikke er brukt opp ennå. Hvis motsatt er tilfelle vil ikke sykmeldingen gå gjennom ( avises ) og da vel heller ikke varsel genereres. Som jeg nevnte på stand-up må vi ta en prat med Lisa til uken om det er greit å gjøre det på denne måten, men nå blir i hvert fall ikke nytt varsel sendt før det har gått 13 uker.

Reply:
Ja, nettopp. Jeg var ikke klar over at sykmeldingen da ble avvist.